### PR TITLE
[Merged by Bors] - feat: add `Metric.Bounded.vadd`

### PR DESCRIPTION
**Pull request by urkud**
Add `Metric.Bounded.smul` and `Metric.Bounded.vadd`, rename the old `smul` to `smul₀`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by ADedecker** [comment]
Could you add a comment on `Metric.Bounded.smul` along the lines of "This result is about isometric actions, see `Metric.Bounded.smul₀` for the result for `NormedSpace`s". I could very well imagine searching for the `NormedSpace` lemma and searching for this exact name.

File: Mathlib/Analysis/NormedSpace/Pointwise.lean [hunk 1]
```diff
@@ -112,14 +112,9 @@ theorem smul_closedBall' {c : 𝕜} (hc : c ≠ 0) (x : E) (r : ℝ) :
...
 
-theorem Metric.Bounded.smul {s : Set E} (hs : Bounded s) (c : 𝕜) : Bounded (c • s) := by
-  obtain ⟨R, hR⟩ : ∃ R : ℝ, ∀ x ∈ s, ‖x‖ ≤ R := hs.exists_norm_le
-  refine' bounded_iff_forall_norm_le.2 ⟨‖c‖ * R, fun z hz => _⟩
-  obtain ⟨y, ys, rfl⟩ : ∃ y : E, y ∈ s ∧ c • y = z := mem_smul_set.1 hz
-  calc
-    ‖c • y‖ = ‖c‖ * ‖y‖ := norm_smul _ _
-    _ ≤ ‖c‖ * R := mul_le_mul_of_nonneg_left (hR y ys) (norm_nonneg _)
-#align metric.bounded.smul Metric.Bounded.smul
+theorem Metric.Bounded.smul₀ {s : Set E} (hs : Bounded s) (c : 𝕜) : Bounded (c • s) :=
```
Comment (urkud) [inline Mathlib/Analysis/NormedSpace/Pointwise.lean:115]:
```suggestion
/-- Image of a bounded set in a normed space under scalar multiplication by a constant is
bounded. See also `Metric.Bounded.smul` for a similar lemma about an isometric action. -/
theorem Metric.Bounded.smul₀ {s : Set E} (hs : Bounded s) (c : 𝕜) : Bounded (c • s) :=
```

File: Mathlib/Topology/MetricSpace/IsometricSMul.lean [hunk 2]
```diff
@@ -425,6 +425,11 @@ theorem nndist_div_left [Group G] [PseudoMetricSpace G] [IsometricSMul G G]
 
 namespace Metric
 
+@[to_additive]
```
Comment (urkud) [inline Mathlib/Topology/MetricSpace/IsometricSMul.lean:428]:
```suggestion
/-- If `G` acts isometrically on `X`, then the image of a bounded set in `X` under scalar
multiplication by `c : G` is bounded. See also `Metric.Bounded.smul₀` for a similar lemma about
normed spaces. -/
@[to_additive "Given an additive isometric action of `G` on `X`, the image of a bounded set in `X`
under translation by `c : G` is bounded"]
```

**Comment by urkud** [comment]
> Could you add a comment on `Metric.Bounded.smul` along the lines of "This result is about isometric actions, see `Metric.Bounded.smul₀` for the result for `NormedSpace`s". I could very well imagine searching for the `NormedSpace` lemma and searching for this exact name.

I added some comments. Feel free to edit them.

**Comment by ADedecker** [comment]
The docstrings are perfect, thanks.
maintainer merge

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**